core/src/main/java/org/apache/metamodel/insert/AbstractRowInsertionBuilder.java

```diff
@@ -44,6 +44,12 @@ public AbstractRowInsertionBuilder(U updateCallback, Table table) {
         _table = table;
     }
 
+    public AbstractRowInsertionBuilder(final U updateCallback, final Table table, final List<Column> columns) {
```

Comment:
I don't think this constructor will be needed if we resolve my first discussion point. And because of that, I prefer to not add it, because not having it keeps the check in place that we don't suddenly start requiring a different method signature wherever this constructor is called.

Reply:
This constructor has been removed.